src/routes/users.js

```diff
@@ -12,19 +14,14 @@ const getUser = async (req, res) => {
 }
 const updateUser = () => {}
 const deleteUser = () => {}
-const checkId = (req, res, next , val ) => {
+const checkId = (req, res, next, val) => {
     console.log(`user id is ${val}`)
     next()
 }
 
 router.param('id', checkId)
-router.route('/')
-.get(fetchUsers)
-.post(createUsers)
+router.route('/').get(fetchUsers).post(createUsers)
 
-router.route('/:id')
-.get(getUser)
-.patch(updateUser)
-.delete(deleteUser)
+router.route('/:id').get(getUser).patch(updateUser).delete(deleteUser)
```

Comment:
curious about this change. does the linter reformat this by default or this is by personal preference?

Reply:
the user.js file serves as a template for how the various endpoint should be built

Reply:
okay cool but still curious as to the change from the multi-line to one-line. wanted to know if it was reformatted by the linter

Reply:
no, it wasn't , it was a personal choice

Reply:
okay cool